src/cls/tabular/cls_tabular_utils.h

```diff
@@ -528,10 +535,14 @@ std::vector<std::string> colnamesFromSchema(schema_vec &schema);
 // the below are used in our root table
 typedef vector<uint8_t> delete_vector;
 typedef const flatbuffers::Vector<flatbuffers::Offset<Record>>* row_offs;
+typedef const flatbuffers::Vector<flatbuffers::Offset<Record_FBU>>* row_offs_fbu_rows;
+typedef const Tables::Cols_FBU* cols_fbu ;
 
 // the below are used in our row table
 typedef vector<uint64_t> nullbits_vector;
 typedef flexbuffers::Reference row_data_ref;
+typedef const flatbuffers::Vector<flatbuffers::Offset<void> >* row_data_ref_fbu_rows ;
```

Comment:
would be great if we could use something like this in our skyrec struct, so that we do not need custom versions.  let's look into that later.